schema.sql

```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
+CREATE DATABASE vet_clinic;
+
 CREATE TABLE animals (
-    name varchar(100)
+    id BIGSERIAL NOT NULL,
+    name VARCHAR(50) NOT NULL,
+    date_of_birth DATE NOT NULL,
+    escape_attempts INT NOT NULL,
+    neutered BOOLEAN NOT NULL,
+    weight_kg DECIMAL NOT NULL
 );
```

Comment:
- [ ] You have name column two times 😨 Kindly remove one of them 😊
- [ ] It would be very nice if just id should not be NOT NULL for now 🤔 So your query should look something similar to below one ⬇️ 👍

```sql
CREATE TABLE animals (
    id BIGSERIAL NOT NULL,
    name VARCHAR(50),
    date_of_birth DATE,
    escape_attempts INT,
    neutered BOOLEAN,
    weight_kg DECIMAL
);
```